manifests/agent.pp

```diff
@@ -12,6 +12,8 @@
   $config_group                 = 'root',
   $config_mode                  = '0644',
   $env                          = $::env,
+  $puppet_run_interval          = '1h',
```

Comment:
If you want to specify the interval, why not run out of cron as opposed to the service?